oak-blob-plugins/src/main/java/org/apache/jackrabbit/oak/plugins/blob/datastore/OakFileDataStore.java

```diff
@@ -139,12 +138,13 @@ public void addMetadataRecord(InputStream input, String name)
 
         try {
             File file = new File(getPath(), name);
-            FileOutputStream os = new FileOutputStream(file);
-            try {
+            try (FileOutputStream os = new FileOutputStream(file)) {
                 IOUtils.copyLarge(input, os);
             } finally {
-                Closeables.close(os, true);
-                Closeables.close(input, true);
+                try {
```

Comment:
Shouldn't we use `IOUtils.closeQuietly` here too, since we are not doing anything with exception?

Reply:
yep